### DISCUSSION TITLE
SSL error

### DISCUSSION THREAD

**Discussion by hankhank10** [dahlia/wikidata#55] [Q&A]
After installing and running the demonstration code I get the following error after the first two lines

`urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1123)>`

**Comment by dahlia** [comment]
What does the following Python script print on your machine?

```python
from urllib.request import urlopen
r = urlopen('https://www.wikidata.org/')
print(r.code, r.reason)
```

**Comment by aGGre55or** [comment]
Install the latest version of OpenSSL on your computer and add "ssl" module for python.
https://slproweb.com/download/Win64OpenSSL-3_2_0.msi
`python -m pip install ssl`
